### PR TITLE
Added option to eat it with a mustard.

### PR DESCRIPTION
**Pull request by pdalak**
Pridana moznost s horcicou.

Added option to eat it with a mustard.

### CODE REVIEW

**Comment by pmallory** [comment]
@casey-collab Can you review this PR please?

File: slovak/horcica/hladam-horcicu.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Však bez horčice to nie je ono. Kyslá rybka jednoznačne len z klasickou plnotučnou horčicou zo skleneného pohára.
```
Comment (casey-collab) [inline slovak/horcica/hladam-horcicu.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by pdalak** [comment]
Should be OK now.

**Pull request closed without merging**